aptos-move/framework/src/natives/aggregator_natives/context.rs

```diff
@@ -198,19 +239,51 @@ mod test {
 
     #[test]
     fn test_into_change_set() {
-        let context = NativeAggregatorContext::new([0; 32], &EmptyStorage);
+        let context = NativeAggregatorContext::new([0; 32], &EmptyStorage, true);
         use AggregatorChange::*;
 
-        test_set_up(&context);
+        test_set_up(&context, true);
         let AggregatorChangeSet { changes } = context.into_change_set();
 
         assert!(!changes.contains_key(&aggregator_id_for_test(100)));
         assert_matches!(changes.get(&aggregator_id_for_test(200)).unwrap(), Write(0));
         assert!(!changes.contains_key(&aggregator_id_for_test(300)));
         assert_matches!(changes.get(&aggregator_id_for_test(400)).unwrap(), Write(0));
         assert_matches!(changes.get(&aggregator_id_for_test(500)).unwrap(), Delete);
-        assert!(changes.contains_key(&aggregator_id_for_test(600)));
-        assert!(changes.contains_key(&aggregator_id_for_test(700)));
+        let _delta_100 = DeltaOp::new(DeltaUpdate::Plus(100), 600, 100, 0);
```

Comment:
nit: `_something` is for unused variables?

Reply:
Done.